recipes/gaitsetpy/meta.yaml

```diff
@@ -40,7 +39,6 @@ test:
   commands:
     - pip check
   requires:
-    - python {{ python_min }}
```

Comment:
Why did you remove these? You had to remove only the part indicated in the review.